timetable/views.py

```diff
@@ -74,7 +74,7 @@ def post(self, request):
         optional_course_subsets = [subset for subset_size in range(max_optional, -1, -1)
                                    for subset in itertools.combinations(optional_courses,
                                                                         subset_size)]
-
+        # TODO remove orphan events if all references are deleted
```

Comment:
By orphan event do you mean `PersonalEvent` rows in the DB? What do you mean by references